pagebuilder/builder.go

```diff
@@ -255,6 +255,15 @@ func (b *Builder) Configure(pb *presets.Builder, db *gorm.DB, l10nB *l10n.Builde
 				dmb = templateM
 			}
 			obj = dmb.NewModel()
+			// Page's PrimaryColumnValuesBySlug will split the slug to get ID, version and locale.
+			// If split failed, will jump to 404 page.
+			splitCount := 2
```

Comment:
Please try to decode the composed id by `presets.SlugDecoder` and check if the db field ID value is an integer